probe/ssh/ssh.go

```diff
@@ -60,12 +67,26 @@ func (s *SSH) Config(gConf global.ProbeSettings) error {
 		return fmt.Errorf("password or private key is required")
 	}
 
+	if len(s.BastionID) > 0 {
+		if bastion, ok := BastionMap[s.BastionID]; ok {
+			log.Debugf("[%s / %s] - has the bastion [%s]", s.ProbeKind, s.ProbeName, bastion.Host)
+			s.bastion = &bastion
+			if err := s.bastion.ParseHost(); err != nil {
```

Comment:
The same bastion endpoint could be parsed repeatedly if it was referenced multiple times. Although it isn't a bug, it's better to fix it.

Reply:
this is a good finding,  and fixed it